Packages/ohif-measurement-table/client/utils/request.js

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
We already have a function that does this. Please switch this to use  

https://github.com/OHIF/Viewers/blob/885dd9da2d0295290a61b9b591a4238387e88725/Packages/ohif-dicomweb-client/src/makeRequest.js

Reply:
I checked out the function but it does not feed all our need in this case. 
Missing things: 
- responseType as arraybuffer on GET method. 
- able to perform POST sending body as well. 

Question: Is that safe for us to be using fetch() as its not supported by IE11 ? I faced that when I was adding my request.js file.

Reply:
Yep we are polyfilling fetch with the 'isomorphic-fetch' package (https://github.com/OHIF/Viewers/blob/885dd9da2d0295290a61b9b591a4238387e88725/Packages/ohif-dicomweb-client/src/makeRequest.js#L3)

Should be easy to add POST and response.arraybuffer() to the existing function.